content/getting-started/onboarding-wizard.md

```diff
@@ -76,16 +71,15 @@ It can take up to a few hours for data to populate in the S3 bucket.
 ### 2. Configure CloudWatch Data Collection
 
 CloudWisdom collects performance data to power analyses. The elements enabled by default are recommended and can be changed at any time.
+
 1. Approve the list of enabled default collections; uncheck any you wish to disable.
 2. Select **Next: Cost and Billing**.
 
 ### 3. Configure Cost & Billing Settings
 
-1. Leave **Cost Explorer** enabled.
-2. Complete the following fields:
-   - **Detailed Billing Source**: select **Cost and Usage Report (recommended)** for your Detailed Billing Source. Detailed Billing Files are a legacy feature of AWS and may not be supported in the future.
-   - **S3 Bucket Name**: Select the s3 bucket you created in [section 1, part 2](/getting-started/onboarding-wizardv2/#2-enable-cost-usage-reports-in-aws).
-   - **Report Path Prefix**: Input the path prefix of an existing CUR or the new one created in section 1, part 2. `CostAndUsageReports/HourlyCSVWithResourceIDs`
+1. Leave **Cost Explorer API** enabled.
+2. Select a **Report Path Prefix**. This path should be connected to the S3 bucket where Cost and Usage Report files are stored.
```

Comment:
```suggestion
2. Select a **Report Path Prefix**. The values listed are a combination of the S3 bucket name where the billing files reside followed by the full Report path prefix of the Cost & Usage report you configured.
```

Reply:
If we don't have access to the CUR endpoint then this selection doesn't work and it reverts back to the old manual way. Should we keep the old instructions that were removed in case we don't have that access? I'm afraid a user may get confused with no other recourse in the docs if they git presented the two new fields